packages/react-chat/src/browser/index.tsx

```diff
@@ -13,16 +11,6 @@ document.body.appendChild(rootEl);
 
 const root = createRoot(rootEl);
 
-const fetchAssistant: Listeners.MessageListener<PostMessage.Type.FETCH_ASSISTANT> = {
-  type: PostMessage.Type.FETCH_ASSISTANT,
```

Comment:
this message used to be the inverse - the widget would wait for the iframe to fetch this and receive it.